packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -41,6 +42,13 @@ class BuildApkCommand extends BuildSubCommand {
   @override
   final String name = 'apk';
 
+  @override
+  Future<Set<DevelopmentArtifact>> get requiredArtifacts async => <DevelopmentArtifact>{
```

Comment:
Assemble doesn't have any required artifacts, so these must be declared in the build_* commands to be up to date. I'm not sure if androidMaven actually needs to be included though, or if gradle will download them itself.

Reply:
I think it doesn't need to.